api-catalog-ui/frontend/src/components/Dashboard/Dashboard.jsx

```diff
@@ -71,6 +73,26 @@ export default class Dashboard extends Component {
         closeAlert();
     };
 
+    /**
+     * This method is used only in the portal in combination with CSS, to scroll the tiles.
+     * @param e
+     */
+    dashboardTileScroll = (e) => {
```

Comment:
I would add apiPortalEnabled check here to be sure this code will not run and it would be better readable as well